services/common/src/components/forms/SteppedForm.tsx

```diff
@@ -110,8 +110,9 @@ const SteppedForm: FC<SteppedFormProps> = ({
       if (handleSaveData && (await saveCheck())) {
         await handleSaveData(getValues(formValues), tab, `${tabs[tabIndex]}`);
       }
-      if (errors.length > 0) return;
-      setTabIndex(indexOf(tabs, tab));
+      if (errors.length === 0 && tab) {
+        setTabIndex(indexOf(tabs, tab));
+      }
```

Comment:
making sure that the "setIsSubmitting" in the finally gets called instead of returning from the function. 
There was something going on where I think the form was reset, I would navigate to the next page, and then it would _never_ let me go to the next page.